backend_client.cc

```diff
@@ -0,0 +1,70 @@
+#include "backend_client.h"
+#include <iostream>
+#include <memory>
+#include <string>
+#include <thread>
+
+#include <gflags/gflags.h>
+#include <grpcpp/grpcpp.h>
+
```

Comment:
Nitpicky, but I just referred to the style guide and I think there shouldn't be a blank line here.